examples/source/1.components/amp-sidebar.html

```diff
@@ -42,102 +44,178 @@
     }
     .sample-sidebar li,
     nav[toolbar] li {
-      margin-bottom: var(--space-2);
-      margin-left: 0;
-      margin-right: var(--space-2);
+      margin: var(--space-2);
+      margin-left: var(--space-4);
       list-style: none;
     }
+    amp-sidebar ul, nav[toolbar] ul {
+      display: block;
+      padding-left: 0;
+    }
     .previewOnly {
       margin: var(--space-2);
     }
     #sidebar-right nav.amp-sidebar-toolbar-target-shown {
         display: none;
     }
+    button {
+      margin-left: var(--space-2);
+    }
+    header {
+      background-color: lightgray;
+      padding-bottom: var(--space-2);
+    }
+    .hamburger {
+      top: 0;
+      right: 0;
+      position: absolute;
+    }
+    h1 {
+      padding: var(--space-1);
+      margin: 0;
+      font-size: 2em;
+    }
+    .desktop-sidebar {
+      background-color: lightblue;
+    }
+    @media (min-width: 783px) {
+      .hamburger {
+        display: none;
+      }
+      main {
+        display: flex;
+        flex-direction: row;
+      }
+      aside {
+        width: 300px;
+      }
+    }
   </style>
 </head>
 <body>
+  <!-- Header -->
+  <header>
+    <h1>Header</h1>
+    <button
+      class="hamburger"
+      on='tap:sidebar-desktop.toggle'
+      aria-label="Click to open sidebar">
+      =
+    </button>
+  </header>
+  
+  <!-- Main Content -->
```

Comment:
Nesting comments is not supported. Suggestion: remove

Reply:
I have removed, thanks!